src/brasil/gov/tiles/upgrades/v4004/__init__.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+from brasil.gov.tiles.config import PROJECTNAME
+from collective.cover.controlpanel import ICoverSettings
+from collective.cover.interfaces import ICover
+from plone import api
+
+import json
+import logging
+
+
+OLD_TILE = u'poll'
+NEW_TILE = u'collective.polls'
+
+
+def _change_tile_type(layout, is_child=False):
+    """Recursivelly change layout type."""
+    if not is_child:
+        layout = json.loads(layout)
+    fixed_layout = []
+    for row in layout:
+        fixed_row = row.copy()
+        old_type = fixed_row.get(u'tile-type', None)
+        if old_type == OLD_TILE:
+            fixed_row[u'tile-type'] = NEW_TILE
+        if u'children' in fixed_row:
+            fixed_row[u'children'] = _change_tile_type(fixed_row[u'children'], True)
+        fixed_layout.append(fixed_row)
+    if is_child:
+        return fixed_layout
+    else:
+        fixed_layout = json.dumps(fixed_layout)
+        return fixed_layout.decode('utf-8')
+
+
+def rename_poll_tile(context):
+    """Rename poll tile"""
+    logger = logging.getLogger(PROJECTNAME)
+    tiles = api.portal.get_registry_record('plone.app.tiles')
+    if OLD_TILE in tiles:
+        tiles.remove(OLD_TILE)
+        tiles.append(NEW_TILE)
+        api.portal.set_registry_record('plone.app.tiles', tiles)
+    record = dict(interface=ICoverSettings, name='available_tiles')
+    available_tiles = api.portal.get_registry_record(**record)
+    if OLD_TILE in available_tiles:
+        available_tiles.remove(OLD_TILE)
+        available_tiles.append(NEW_TILE)
```

Comment:
this has to happen eve if `OLD_TILE` is not in `available_tiles`.

Reply:
this field is used to set the enabled tiles (the name is wrong), so this if is telling: If the pool tile is being used in the portal, it will be enabled.